third_party/terraform/tests/resource_compute_backend_service_test.go.erb

```diff
@@ -1508,3 +1532,47 @@ resource "google_compute_http_health_check" "zero" {
 }
 `, serviceName, sampleRate, checkName)
 }
+
+<% unless version == 'ga' -%>
+func testAccComputeBackendService_regionNegBackend(suffix string) string {
+	return fmt.Sprintf(`
+resource "google_compute_backend_service" "backend" {
+  name                            = "backend%s"
```

Comment:
this should be prefixed with tf-test so it gets swept (probably the other resources too)